### ISSUE TITLE
Router not starting: more debug info witih -vvv

### ISSUE THREAD

**Issue by tacman** [voryx/ThruwayBundle#93]
Using Symfony 4.2, I've followed the installation instructions, the service is registered fine:
```
 bin/console thruway:debug
 URI             Type Worker  File                                               Method    
 com.example.add RPC  default /home/tac/rappnet/src/Controller/AppController.php addAction 
```

but when I go to start the router, worker or process, it stops with no information:

```
bin/console thruway:router:start -vvv
Making a go at starting the Thruway Router
```

Nothing else is listening on 8080 or 8081, I'm using the default configuration for everything.  Any suggestions on how to debug this?

**Comment by davidwdan** [comment]
What version of the Thruway Bundle and PHP are you using?

**Comment by tacman** [comment]
I added "enable_logging: true" to voyrx.yaml and figured out my problem.  First, "making a go..."  sounds like it's trying to start but is unsuccessful.  It was actually working, but I tested using autobahn, and it was connecting to port 9000, and I didn't look carefully enough (until I got the logs).  Looks like it's working now!

PHP 7.2

thruway/client                           0.5.4              Thruway WAMP client
thruway/pawl-transport                   0.5.0              Pawl WebSocket Transport for Thruway Client
thruway/ratchet-transport                0.5.0              Ratchet Transport for Thruway
voryx/thruway                            0.5.3              Thruway WAMP router core
voryx/thruway-bundle                     0.4.1              WebSockets (WAMP2) integration for Symfony2
voryx/thruway-common                     1.0.5              WAMP2 Common Library for Thruway
